model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/RealmCacheSession.java

```diff
@@ -537,40 +537,8 @@ static String getRoleByNameCacheKey(String container, String name) {
     }
```

Comment:
Just checking here - we are removing any caching of client lists, and only having caching of individual clients?

Reply:
Yes, that was the idea. I thought we shouldn't cache the pages when it comes to getting clients, but if you think this should change, let me know!

Reply:
I was mostly just checking as we shouldn't cache clients at all. Client lists are only used in admin console, and while being used in admin console often changed. So it doesn't make any sense to cache client lists.